src/ajax/parseXML.js

```diff
@@ -9,7 +9,8 @@ jQuery.parseXML = function( data ) {
 		return null;
 	}
 
-	// Support: IE9
+	// Support: IE 9-11 only
```

Comment:
It's not, though. To me, the `<=` implies we are writing code that deals with those browsers, but we aren't.